docs/user/guides/setup-single-sign-on-google-email.rst

```diff
@@ -0,0 +1,79 @@
+How to setup Single Sign-On (SSO) with Google Workspace
+=======================================================
+
+.. include:: /shared/admonition-rtd-business.rst
+
+This how-to guide will provide instructions on how to enable :abbr:`SSO (Single Sign-on)` with Google Workspace.
+If you want more information on this feature,
+please read :doc:`/commercial/single-sign-on`
+
+Prerequisites
+-------------
+
+Organization permissions
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To change your :ref:`Organization <commercial/organization:Organizations>`'s settings,
+you need to be an *owner* of that organization.
+
+You can validate your ownership of the Organization with these steps:
+
+* Navigate to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Organization name>`
+* Look at the **Owners** UI elements on the right menu.
+
+If you'd like to to modify this setting and are not an owner,
+you can ask an existing organization owner to follow this guide.
+
+User setup
+~~~~~~~~~~
+
+Using this setup,
+all users who have access to the configured Google Workspace will be granted a subset of permissions on your organization automatically on account creation.
+.. tip::
+
+   You can still add outside collaborators and manage their access. Organizations may for instance wish to give read-access to onboarding documents or grant temporary write access to external consultants.
+
+.. seealso::
+
+   :doc:`/commercial/sharing`
+     Read more about methods for sharing **read** access to private sets of documentation without requiring a login for Read the Docs or your SSO provider.
+
+Enabling SSO
+------------
+
+By default, users that sign up with a Google account do not have any permissions over any project.
+However, you can define which teams users matching your company's domain email address will auto-join when they sign up.
+
+You can enable this feature in your organization:
+
+* Navigate to :guilabel:`<Username dropdown>` > :guilabel:`Organizations` > :guilabel:`<Organization name>` > > :guilabel:`Settings` > :guilabel:`Authorization`
+* Select :guilabel:`Google` as the *Provider*
+* Specify your Google Workspace domain as the *Domain*.
```

Comment:
We should make usage of `guilabel` here as well. BTW, did we already agreed on this? I thought we did, but I may be confused.

Reply:
Looks like Sphinx actually has `:menuselection:`! Perhaps we should also be using this? https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-menuselection

Reply:
Looks good, yeah. I saw that @benjaoming mentioned in the style guide PR as well 👍🏼

Reply:
I've never used it, so I'm not sure how it renders or even if our theme supports it 😄

Reply:
The only thing `:menuselection:` doesn't help us with is the seperator. Sphinx docs suggest `>` and I think that sounds good (but then they use `-->` in the example which seems oddly long).

Reply:
> I've never used it, so I'm not sure how it renders or even if our theme supports it 

If it doesn't support it, we are currently doing new releases, so we should be able to add an issue and fix that quite quickly :)

Reply:
This is addressed in the style guide now 👍

Reply:
This is how the two render:

<img width="661" alt="Screenshot 2022-12-15 at 3 08 32 PM" src="https://user-images.githubusercontent.com/25510/207985949-c40b050a-40ec-4b55-bedf-ad9367f3358a.png">

Basically `-->` gets a nicer glyph. Both aren't styled though, and should probably be styled as the same as `:guilabel:` 🙃